Resources/Prototypes/Entities/Objects/Misc/revflash.yml

```diff
@@ -0,0 +1,43 @@
+- type: entity
+  name: flash
+  parent: BaseItem
+  id: RevFlash
+  description: An ultrabright flashbulb with a trigger, which causes the victim to be dazed and... wait, this seems to have a strange compartment on the bottom.
```

Comment:
imo rev flashes shouldnt be special in any way from regular flashes, idk if this prototype needs to exist (if it does need to exist can it just parent from the flash entity instead of duplicating like this?)

Reply:
Yeah you're right, I thought 5 charges might've been too little so I made this but now that sci can make flashes I'm gonna just remove it.